### PR TITLE
Added Test based on Veronique's test case, plus fixed several resulting errors

### PR DESCRIPTION
**Pull request by saulakravitz**

### CODE REVIEW

File: VRDR/DeathRecord.cs [hunk 1]
```diff
@@ -1334,17 +1334,18 @@ public string CertificationRoleHelper
...
                 }
                 return null;
             }
             set
             {
                 if ((value != null) && !VRDR.Mappings.CertifierTypes.FHIRToIJE.ContainsKey(value))
                 { //other
-                    if (DeathCertification == null)
-                    {
-                        CreateDeathCertification();
```
Comment (smacadam) [inline VRDR/DeathRecord.cs:1346]: why was this call removed?

File: VRDR/IJEMortality.cs [hunk 2]
```diff
@@ -3751,11 +3764,24 @@ public string TRANSPRT
...
             {
-                return Get_MappingFHIRToIJE(Mappings.MannerOfDeath.FHIRToIJE, "TransportationRole", "TRANSPRT");
+                var ret = record.TransportationRoleHelper;
+                if(Mappings.TransportationIncidentRole.FHIRToIJE.ContainsKey(ret)){
+                    return Get_MappingFHIRToIJE(Mappings.TransportationIncidentRole.FHIRToIJE, "TransportationRole", "TRANSPRT");
+                }
+                else
+                {
+                    return ret;  // If the return value is not a code, it is just an arbitrary string, so return it.
+                }
```
Comment (smacadam) [inline VRDR/IJEMortality.cs:3774]: Could this check be moved int Get_MappingFHIRToIJE?

File: VRDR/DeathRecord.cs (hunk 1 above)
Comment (saulakravitz) [reply VRDR/DeathRecord.cs:1346]: It is redundant with the code in on line 1299....

File: VRDR.Tests/DeathRecord_Should.cs [hunk 3]
```diff
@@ -3150,6 +3159,43 @@ public void Test_GetDemographicCodedContentBundle()
...
+            DeathRecord record = ije.ToDeathRecord();
+            DeathRecord record1 = new DeathRecord(record.GetCauseOfDeathCodedContentBundle(), false);
+            IJEMortality ije2 = new IJEMortality(record);
+            Assert.Equal("DDDD", ije2.CERTL);
+            Assert.Equal("I219", ije2.MAN_UC);
+            Assert.Equal("Y",ije2.AUTOP);
+            Assert.Equal( "Y",ije2.AUTOPF);
+            Assert.Equal("Hover Board Rider", ije.TRANSPRT);
+            // THIS TEST FAILS
+            // Assert.Equal( "21I219  31I251  61E119  62F179  63I10  64E780",ije2.EAC);
```
Comment (smacadam) [inline VRDR.Tests/DeathRecord_Should.cs:3196]: the commented out test on line 3196 still fails? does it need a new ticket to address the issue?

**Review by smacadam — APPROVED** [review]

**Pull request merged**